### ISSUE TITLE
Creating a quiz does not work

### ISSUE THREAD

**Issue by YassineLambarki** [SkyCascade/SkyLearn#2]
**Describe the bug**
Creating a quiz does not work.

**To Reproduce**
Access to http://localhost:8000/admin/quiz/quiz/add/
`Key 'title_en' not found in 'QuizForm'. Choices are: questions.`

**Expected behavior**
Successful creation of quiz.
Know that other types of questions work.

**Screenshots**
![image](https://github.com/user-attachments/assets/b37dac1d-5f62-4263-a795-75c7866ea35f)

**Comment by adilmohak** [comment]
@YassineLambarki It seems like you haven't migrated all the migrations. Lang support is recently included, run the migration command and let us know if the bug persists.

`python manage.py migrate`

**Comment by YassineLambarki** [comment]
Still occuring 
![image](https://github.com/user-attachments/assets/5151ad2a-0f23-44fa-b525-6bab9ae7581c)

**Comment by adilmohak** [comment]
@YassineLambarki Could you please check now

**Comment by YassineLambarki** [comment]
No, https://stackoverflow.com/questions/48842889/django-templatesyntaxerror-invalid-filter

**Comment by adilmohak** [comment]
I just double-checked, and it is working on my side. Did you rebase or pull the latest changes? Also is this error happen on the same page `admin/quiz/quiz/add`?

**Comment by sanyaade-teachings** [comment]
> Access to http://localhost:8000/admin/quiz/quiz/add/

I had this error when using **localhost:8000** but no errors if you use **127.0.0.1:8000**
Could the Django MVC issues? I have had the same problems with other Django projects. The settings treat 127.0.0.1 differently from localhost
Google: Django localhost vs 127.0.0.1  to learn more about this

Hope this helps!

God blesses!!!

Regards
